### ISSUE TITLE
Style editor tab

### ISSUE THREAD

**Issue by AGlass0fMilk** [lvgl/lv_gui_builder#4]
Since most of the visual configuration in lvgl is done using `style` objects, perhaps we can have a dedicated tab in the GUI builder where all the styles in a project can be created and configured. 

This would give the user a complete picture of how many styles they use in their UI and thus the memory usage they are incurring.

If this should be a feature, what other features will this tab have? Can you select a style and preview different widgets with that style?

Please comment with your ideas.

**Comment by kisvegabor** [comment]
Good idea to have a style editor tab where you can see and configure all your styles. I agree, it also would be nice to add some simple objects for `body`, `text`, `line` and `image` properties.

Besides style editor two it would be great to have
- Font tab: 
  - Add ttf files
  - Set what sizes (and other parameters) are required from them 
  - Automatically generate/update the fonts when the code is exported
- Image tab
  - Add images (bmp, jpg, png)
  - Set some conversion properties (true color, true color alpha etc. same as in the image converter tool) 
  - Automatically generate/update the images when the code is exported

**Comment by kisvegabor** [comment]
@AGlass0fMilk did you have a chance to make further experiments?

**Comment by AGlass0fMilk** [comment]
I haven't had time to work on it since we decided to switch to Qt.

What I have so far is a mocked up a main window based on the GTK builder (Glade) with separate tabs for each of these utilities.

I'll try to find some time soon to work on this.

**Comment by kisvegabor** [comment]
Ok, thank you for the feedback! :+1:
